cmd/controller-manager/app/options/generic.go

```diff
@@ -103,9 +103,7 @@ func (o *GenericControllerManagerConfigurationOptions) Validate(allControllers [
 		if controller == "*" {
 			continue
 		}
-		if strings.HasPrefix(controller, "-") {
-			controller = controller[1:]
-		}
+		strings.TrimPrefix(controller, "-")
 		if !allControllersSet.Has(controller) {
```

Comment:
does this fix necessary?
if necessary, IMO，we should change like this:
```
controller=strings.TrimPrefix(controller, "-")
```
but in this way, the cost is greater than change before.
@baasbank WDYT?

Reply:
ah thanks, will fix shortly

Reply:
fixed